driver/src/main/java/oracle/nosql/driver/http/Client.java

```diff
@@ -717,9 +717,8 @@ public Result execute(Request kvRequest) {
                 throw new NoSQLException("Unexpected exception: " +
                         rae.getMessage(), rae);
             } catch (InvalidAuthorizationException iae) {
-                /* allow a single retry on clock skew errors */
-                if (iae.getMessage().contains("clock skew") == false ||
-                    kvRequest.getNumRetries() > 0) {
+                /* allow a single retry on clock skew / auth errors */
```

Comment:
maybe comment that these are errors indicating actual invalid/expired auth info vs a permission-related error